entrypoint.sh

```diff
@@ -57,29 +61,85 @@ function parseInputs {
     exit 1
   fi
 
+  if [[ "${INPUT_SCEPTRE_PIPFILE}" != "" ]] \
+     && [[   "${INPUT_SCEPTRE_REQUIREMENTS}" != "" \
+          || "${INPUT_SCEPTRE_PLUGINS}" != "" \
+          || "${INPUT_SCEPTRE_TROPOSPHERE_VERSION}" != "" \
+          || "${INPUT_SCEPTRE_VERSION}" != "" ]]; then
+    echo "WARNING: Detected Pipfile and at least one other method for specifying"
+    echo "         dependencies/versions. Only the Pipfile will be installed."
+    echo "         Consider only specifying the dependencies/versions in the Pipfile."
+  fi
+
+  if [[ "${INPUT_SCEPTRE_PIPFILE}" != "" || "${INPUT_SCEPTRE_REQUIREMENTS}" != "" ]]; then
+
+    if [[ "${INPUT_SCEPTRE_PLUGINS}" != "" ]]; then
+        echo "WARNING: Detected Pipfile and/or requirements file and Sceptre plugins."
+        echo "         Consider only specifying the Sceptre plugins in either file."
+    fi
+
+    if [[ "${INPUT_SCEPTRE_TROPOSPHERE_VERSION}" != "" ]]; then
+      echo "WARNING: Detected Pipfile and/or requirements file and Troposhere version."
+      echo "         Consider only specifying the Troposphere version in either file."
+    fi
+
+    if [[ "${INPUT_SCEPTRE_VERSION}" != "" ]]; then
+      echo "WARNING: Detected Pipfile and/or requirements file and a Sceptre version."
+      echo "         Consider only specifying the Sceptre version in the Pipfile."
+    fi
+
+  fi
 }
 
 function installDeps {
-  if [[ "${sceptreTroposphere}" == 1 ]]; then
-    echo "Installing Troposphere"
-    pip install --no-input troposphere==$sceptreTroposphereVer
+
+  if [[ "${INPUT_SCEPTRE_PIPFILE}" != "" ]]; then
+    echo "Installing Pipenv"
+    if [[ "${INPUT_SCEPTRE_PIPENV_VERSION}" != "" ]]; then
+      pip install --no-input pipenv==$INPUT_SCEPTRE_PIPENV_VERSION
+    else
+      pip install --no-input pipenv
+    fi
```

Comment:
i'm not sure i like the idea of supporting a separate dependency manager in this action.  I think it's going a bit above the responsibility of a GH action.  At the most i think we should only install dependencies from requirements.txt files.

If you must have pipenv then how about we just add a specific version to the docker image and then in here we only run `pipenv install`?  hmm now that i'm thinking about this what about `pipenv run` which is what pipenv recommends to make sure you are running with dependencies installed by pipenv?